indico/modules/events/timetable/client/js/UnscheduledContributions.tsx

```diff
@@ -124,7 +123,9 @@ export default function UnscheduledContributions({dt}: {dt: Moment}) {
       text: (
         <div styleName="session">
           <Label empty circular color="grey" />
-          <Translate>No assigned session</Translate>
+          <span>
+            <Translate>No assigned session</Translate>
+          </span>
```

Comment:
Tip: you can pass in tags into the `<Translate>` blocks.

```suggestion
          <Translate as="span">No assigned session</Translate>
```